lib/rules/selector-disallowed-list/README.md

```diff
@@ -115,3 +115,29 @@ The following pattern is _not_ considered a problem:
   .foo {}
 }
 ```
+
+### `ignore: ["keyframe-selectors"]`
+
+Ignore selectors that are keyframe selectors.
+
+Given:
+
+```json
+[/from/, { ignore: ['keyframe-selectors'] }]
```

Comment:
minor

- add missing `"`
- replace `'` by `"`

Reply:
[suggest] Yes, the current snippet is invalid JSON. Should be:

```suggestion
["/from/", { "ignore": ["keyframe-selectors}"] }]
```

Besides, can you fix other secondary option snippets in this README, too? To keep consistency, they should be:

```json
[".foo", { "splitList": true }]
```

```json
[".foo", { "ignore": ["inside-block"] }]
```